data/modules/Scout/Scout.lua

```diff
@@ -303,24 +303,30 @@ local function calcOrbitalScanMission(sBody, difficulty, reward)
 
 	local resolutionScalar = math.invlerp(p.resolution_min, p.resolution_max, resolution)
 
+	local body_coverage = coverage
 	if radiusScalar > 1.0 then
 		-- body is small, increase coverage of the scan
-		coverage = math.min(coverage, 1.0) * (radiusScalar ^ 0.9)
+		body_coverage = math.min(coverage, 1.0) * (radiusScalar ^ 0.9)
 	else
 		-- body is large, reduce coverage of the scan proportionally
-		coverage = math.min(coverage, 1.0) * radiusScalar
+		body_coverage = math.min(coverage, 1.0) * radiusScalar
 		-- similarly increase the resolution of the scan to ensure we can scan at higher altitudes
 		resolution = resolution * (1.0 + math.log(radiusKm / p.nominal_radius, 10))
 	end
 
-	coverage = math.min(coverage * realDifficulty, 1.0)
+	-- adjust coverage against difficulty and clamp to a maximum of 100%
+	body_coverage = math.min(body_coverage * realDifficulty, 1.0)
+
+	-- finally, convert the coverage into km2 and limit the number of decimals
+	local body_area = 4 * math.pi * radiusKm^2
+	local coverageKm2 = math.floor(body_area * body_coverage * 10000) / 10000
```

Comment:
Note that we have `utils.round(val, unit_last_place)`, in this case `utils.round(body_area * body_coverage, 0.0001)`.

Reply:
Fixed in 778e5a38e2cf9c784a046d38e6b55c47bfb3e5f6